src/Security/MemberAuthenticator/CMSMemberLoginForm.php

```diff
@@ -58,15 +58,21 @@ public function getFormFields()
                 'Password',
                 CheckboxField::create(
                     "Remember",
-                    _t('SilverStripe\\Security\\Member.KEEPMESIGNEDIN', "Keep me signed in")
-                )->setAttribute(
-                    'title',
                     _t(
-                        'SilverStripe\\Security\\Member.REMEMBERME',
-                        "Remember me next time? (for {count} days on this device)",
+                        'SilverStripe\\Security\\Member.REMEMBER_ME',
```

Comment:
Use existing translation string rather than creating a new one

Reply:
We are changing the string, so we need to use the new one.

Reply:
Ah right ok, fair enough